orthopairs/src/main/java/org/reactome/release/orthopairs/OrthopairFileGenerator.java

```diff
@@ -0,0 +1,93 @@
+package org.reactome.release.orthopairs;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.json.simple.JSONObject;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.nio.file.StandardOpenOption;
+import java.util.*;
+
+public class OrthopairFileGenerator {
+    private static final Logger logger = LogManager.getLogger();
+    // Create source-target protein mapping file
+    public static void createProteinHomologyFile(String speciesKey, String sourceTargetProteinMappingFilename, JSONObject speciesJSON, Map<String,Set<String>> speciesProteinHomologs ) throws IOException {
+
+        logger.info("\tGenerating " + sourceTargetProteinMappingFilename);
+        File sourceTargetProteinMappingFile = new File(sourceTargetProteinMappingFilename);
+        if (sourceTargetProteinMappingFile.exists()) {
+            sourceTargetProteinMappingFile.delete();
+        }
+        sourceTargetProteinMappingFile.createNewFile();
+
+        List<String> sourceProteinIds = new ArrayList<>();
+        sourceProteinIds.addAll(speciesProteinHomologs.keySet());
+        Collections.sort(sourceProteinIds);
+        for (String sourceProteinId : sourceProteinIds) {
+            Set<String> targetProteinIds = speciesProteinHomologs.get(sourceProteinId);
+            targetProteinIds.remove("LDO");
+            String proteinOrthologLine = sourceProteinId.split("=")[1] + "\t";
```

Comment:
The pattern `String.split("=")[1]` is done a few times in the class.  Maybe it could be a method with a name describing what the value after the equals sign is for explanation and consolidation to one place?